docs/mocksupport.rst

```diff
@@ -414,104 +390,247 @@ of the definition and execution of InvokeMethod.
+:class:`~pywbem.WBEMConnection`, with the following exceptions:
+
+- The filter query related parameters `FilterQuery` and `FilterQueryLanguage`
+  are ignored and no such filtering takes place.
+
+- The `ContinueOnError` parameter is ignored because injecting an error into
+  the processing of the pull operations is not supported by the pywbem mock
+  support, so no failures can happen during the processing of the pull
+  operations.
+
+- The `OperationTimeout` parameter is currently ignored. As a result, there
+  will be no timeout if the
+  :attr:`~pywbem_mock.FakedWBEMConnection.response_delay` property is set to a
+  time larger than the `OperationTimeout` parameter.
+
+The faked pull operations are:
+
+- **OpenEnumerateInstances**: Behaves like
+  :meth:`~pywbem.WBEMConnection.OpenEnumerateInstances`,
+  with the stated exceptions.
+
+- **OpenEnumerateInstancePaths**: Behaves like
+  :meth:`~pywbem.WBEMConnection.OpenEnumerateInstancePaths`,
+  with the stated exceptions.
+
+- **OpenAssociatorInstances**: Behaves like
+  :meth:`~pywbem.WBEMConnection.OpenAssociatorInstances`,
+  with the stated exceptions.
+
+- **OpenAssociatorInstancePaths**: Behaves like
+  :meth:`~pywbem.WBEMConnection.OpenAssociatorInstancePaths`,
+  with the stated exceptions.
+
+- **OpenReferenceInstances**: Behaves like
+  :meth:`~pywbem.WBEMConnection.OpenReferenceInstances`,
+  with the stated exceptions.
+
+- **OpenReferenceInstancePaths**: Behaves like
+  :meth:`~pywbem.WBEMConnection.OpenReferenceInstancePaths`,
+  with the stated exceptions.
+
+- **OpenQueryInstances**: Behaves like
+  :meth:`~pywbem.WBEMConnection.OpenQueryInstances`,
+  with the stated exceptions.
+
+- **PullInstancesWithPath**: Behaves like
+  :meth:`~pywbem.WBEMConnection.PullInstancesWithPath`,
+  with the stated exceptions.
+
+- **PullInstancePaths**: Behaves like
+  :meth:`~pywbem.WBEMConnection.PullInstancePaths`,
+  with the stated exceptions.
+
+- **PullInstances**: Behaves like
+  :meth:`~pywbem.WBEMConnection.PullInstances`,
+  with the stated exceptions.
+
+- **CloseEnumeration**: Behaves like
+  :meth:`~pywbem.WBEMConnection.CloseEnumeration`,
+  with the stated exceptions.
 
-.. _`Server pull operations methods`:
 
-Server pull operations methods
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. _`Faked iter operations`:
 
-The fake pull operations implement all of the Open and Pull WBEM server
-methods with the same behavior as the real operations except as follows.
+Faked iter operations
+^^^^^^^^^^^^^^^^^^^^^
 
-- Filter Query Language parameters are ignored and the full instance responses
-  generated. Since FQL only filters the set of instanced defined by
-  the corresponding request (i.e. Open/Pull EnumerateInstances simply
-  filters instances that would be acquired by EnumerateInstances. This means
-  that these methods can be used but may return more instances than would
-  be returned without the filters.
+The faked iter operations (like the real iter operations) do not directly
```

Comment:
This one bothers me in that we do not really have faked iter operations but use the real ones.

Reply:
Ah ok. I did not realize that. Changed the text accordingly.